README.md

```diff
@@ -12,24 +12,47 @@
         --schema-handler-package "<LINK_TO_S3>"
     ```
 
-    With the following placeholders:
+2. View the version of the newly registered resource by running the following in your terminal:
+
+    ```shell
+    aws cloudformation list-type-versions \
+    --region "<REGION>" \
+    --type RESOURCE \
+    --type-name "<DATADOG_RESOURCE_NAME>"
+    ```
+
+3. Set this newly registered version as the `default` by running the following in your terminal:
+
+    ```shell
+    aws cloudformation set-type-default-version \
+        --region "<REGION>" \
+        --type RESOURCE \
+        --version-id <VERSION> \
+        --type-name "<DATADOG_RESOURCE_NAME>" \
+    ```
+
+    With the following required placeholders:
     * `<REGION>`: Your AWS region.
-    * `<DATADOG_RESOURCE_NAME>`: The name of the resource to register, refer to the table below to see the supported resources.
+    * `<DATADOG_RESOURCE_NAME>`: The name of the resource to register, refer to the [table below](#resources-available) to see the Datadog supported resources.
     * `<LINK_TO_S3>`: S3 link to the resource.
-      * S3 link: `s3://datadog-cloudformation-resources/<RESOURCE_FOLDER>/<RESOURCE_FOLDER>-<VERSION>.zip`
+      * S3 link: `s3://datadog-cloudformation-resources/<RESOURCE_NAME>/<RESOURCE_NAME>-<VERSION>.zip`
+      * See the [Resources Available section](#resources-available), which links to examples of the latest supported S3 links.
+    * `VERSION`: The underlying version of the resource as returned by the command in step `2`.
```

Comment:
I would rename the `VERSION` in the s3 link to `RESOURCE_VERSION` and the `VERSION` the `set-type-default-version` as `VERSION_ID` to avoid any confusion

Reply:
Good call! I think that'll help avoid confusion. Just pushed that change.